<launch>
  <arg name="results" default="$(env HOME)/Desktop/results.txt"/>

  <rosparam command="load" file="$(find sos_slam)/calibs/Gazebo/calib.yaml" ns="sos_slam"/>

  <node name="sos_slam" pkg="sos_slam" type="sos_slam_node" output="screen" required="true">
    <param name="imu_topic" value="/imu" />
    <param name="cam0_topic" value="/stereo_camera/left/image_raw" />
    <param name="cam1_topic" value="/stereo_camera/right/image_raw" />
    <param name="calib0" value="$(find sos_slam)/calibs/Gazebo/camera0.txt" />
    <param name="calib1" value="$(find sos_slam)/calibs/Gazebo/camera1.txt" />
    <param name="scale_opt_thres" value="5" />
    <param name="results" value="$(arg results)" />
    <param name="weight_imu_dso" value="6" />
    <param name="lidar_range" value="20" />
  </node>
  </launch>